builtin/trap_osh.py

```diff
@@ -346,6 +370,31 @@ def _RemoveTheRest(self, arg_r, allow_legacy=True):
             self.trap_state.RemoveItem(parsed_id)
             arg_r.Next()
 
+    def _IgnoreTheRest(self, arg_r):
```

Comment:
Hm I think we could use `_AddTheRest()` with `command.NoOp` ... that will simplify the code